src/components/text-field/style/lumapps/_index.scss

```diff
@@ -84,6 +84,13 @@
         }
 
         textarea {
+            overflow: hidden;
```

Comment:
Did you check on angularjs version?

Reply:
yes, I compared the css with the ones here:
http://ui.lumapps.com/components/text-field